public/components/trace_analytics/home.tsx

```diff
@@ -431,7 +432,7 @@ export const Home = (props: HomeProps) => {
                   chrome={props.chrome}
                   http={props.http}
                   traceId={decodeURIComponent(traceId)}
-                  mode={mode}
+                  mode={((traceMode as unknown) as TraceAnalyticsMode) || mode}
```

Comment:
Why can the mode in TraceView be potentially different from the mode used throughout this page? It looks like the state used throughout this page is set by sessionStorage and then traceMode here is from the url params?

Reply:
Added it to the other calls. Thanks for noticing this (should be in all calls). 
Essentially if the url contains mode= we know the user is navigation through a shared link or opening a new tab. In these cases we want this to take precedence over the session storage that could potentially be empty or using the default value.

Reply:
Sounds good!